tests/core/offers/test_repository.py

```diff
@@ -3,9 +3,10 @@
 
 import pytest
 
-from pcapi.core.offers import factories
+from pcapi.core.offers import factories as offer_factories
```

Comment:
```suggestion
from pcapi.core.offers import factories as offers_factories
```

C'est sûrement moins correct en anglais, mais plus simple et homogène de se dire que le nom de l'alias est composé du nom du package "offers" (avec un "s") suivi de "_" suivi du nom du module.

(Sinon, j'aime bien juste garder `factories` puisque ce module teste des offres. Mais bon, c'est un peu subjectif.)

Reply:
J'avais renommé pour coller au `user_factories` que j'avais trouvé ailleurs
Je renomme également `user_factories` en `users_factories` ?